pkg/plugins/resources/temurin/spec.go

```diff
@@ -0,0 +1,20 @@
+package temurin
+
+type Spec struct {
+	// ReleaseLine specifies the type of Temurin Release to retrieve: 'lts' (default), feature.
+	ReleaseLine string `yaml:",omitempty"`
+	// ReleaseType specifies the type of Temurin release: 'ga' (default), 'ea' (nightly releases).
+	ReleaseType string `yaml:",omitempty"`
+	// FeatureVersion specifies the Java Feature version (major) to filter results.
+	FeatureVersion int `yaml:",omitempty"`
+	// Result specifies the type of result returned by the resource: 'version' (default), installer_url, checksum/
+	Result string `yaml:",omitempty"`
```

Comment:
I like that you allow different kind of result.
It's something that I should have for other plugins like gittag should be able to return a git tag or hash associated to that tag